From 814e1ffe9c7e0878101f9d16b740983e510d43f3 Mon Sep 17 00:00:00 2001
From: Mooffie <mooffie@gmail.com>
Date: Mon, 5 Dec 2016 01:48:33 +0200
Subject: [PATCH] mcedit: syntax.syntax: highlight comments preceded by spaces.

Comments don't have to start at the 1st column. We add support for this case.
---
 misc/syntax/syntax.syntax | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/misc/syntax/syntax.syntax b/misc/syntax/syntax.syntax
index d7ed50d..fd6d132 100644
--- a/misc/syntax/syntax.syntax
+++ b/misc/syntax/syntax.syntax
@@ -52,3 +52,9 @@ context default
     keyword whole white white
 
 context linestart # \n brown
+    spellcheck
+
+# Ideally we would have defined comments with "context linestart \[\s\]# ...",
+# but contexts can't start with a character class. So we define comments twice.
+context linestart \s\[\s\]# \n brown
+    spellcheck
-- 
2.9.3

